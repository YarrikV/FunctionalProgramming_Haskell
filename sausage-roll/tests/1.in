~ ~ ~s~ ~s~ ~ 
~ +S+n+s+n+s~ 
~ ~x~ ~n~ ~n~ 
