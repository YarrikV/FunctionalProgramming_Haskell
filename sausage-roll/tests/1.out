~ ~ ~s~ ~s~ ~ 
~ +S+n+s+n+s~ 
~ ~x~ ~n~ ~n~ 

~ ~ ~ ~ ~s~ ~ 
~ +E+x+s+n+s~ 
~ ~ ~ ~n~ ~n~ 

~ ~ ~ ~ ~ ~s~ 
~ + +E+x+s+n~ 
~ ~ ~ ~ ~n~ ~ 

~ ~ ~ ~ ~ ~ ~ 
~ + + +E+x+s~ 
~ ~ ~ ~ ~ ~n~ 

